readme.txt

```diff
@@ -74,6 +74,7 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
 == Changelog ==
 
 == Unreleased ==
+- Fix: State `md5` as `@woocommerce/data`'s npm dependency. #7087
```

Comment:
This won't be necessary given the changelog in the data package.
Just add `no changelog necessary as it is in data package` to the PR description in order for the changelog action to pass.